src/ui/map.js

```diff
@@ -481,7 +481,7 @@ class Map extends Camera {
      * If the map's current zoom level is lower than the new minimum,
      * the map will zoom to the new minimum.
      *
-     * @param {number | null | undefined} minZoom The minimum zoom level to set (0-20).
+     * @param {number | null | undefined} minZoom The minimum zoom level to set (0-22).
```

Comment:
I think this should also be zoom 24 to match the minzoom 24 later at the bottom of this PR. Looks like the ship on this one has sailed.